src/views/profile.njk

```diff
@@ -1,19 +1,55 @@
 {% extends "page.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
 
 {% block content %}
-    <h1>{{ heading }}</h1>
-    {% if user %}
-    Authenticated: {{ isAuthenticated }}<br/>
-    Display name: {{user.display_name}}<br/>
-    Email: {{user.email}}
-        <br/>
-    Share requests:
-    <ul>
-            {% for assetId, requestData in requestForms %}
-                <li>
-                    <a href="/acquirer/{{assetId}}/start">{{assetId}}</a>
-                </li>
-            {% endfor %}
-        </ul>
-        {%endif%}
-    {% endblock %}
\ No newline at end of file
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-xl">Account</h1>
+
+      {% if user %}
+      {{ govukTable({
+        captionClasses: "govuk-table__caption--m",
+        firstCellIsHeader: false,
+        rows: [
+          [
+            {
+              text: "Name"
+            },
+            {
+              text: user.display_name
+            }
+          ],
+          [
+            {
+              text: "Email"
+            },
+            {
+              text: user.email
+            }
+          ],
+          [
+            {
+              text: "Organisation"
+            },
+            {
+              text: user.organisation if user.organisation else "unavailable"
+            }
+          ],
+          [
+            {
+              text: "Role"
+            },
+            {
+              text: user.role if user.role else "unavailable"
+            }
+          ]
+        ]
+      }) }}
+
+    {% endif %}
+    </div>
+  </div>
+
+{% endblock %}
```

Comment:
We have lost the share requests here?

Reply:
This is based on the Miro wireframes of the user profile which do not show the share requests:
![image](https://github.com/co-cddo/data-marketplace/assets/107464669/427d6bbc-a389-4e44-baa2-bf8e3dbdcece)